client/rest/src/plugins/metadata/MetadataDb.js

```diff
@@ -78,6 +80,40 @@ class MetadataDb {
 			.toArray()
 			.then(entities => Promise.resolve(this.catapultDb.sanitizer.renameIds(entities)));
 	}
+
+	async binDataByMetalId(metalId) {
+		const compositeHashes = [routeUtils.parseArgument(metal.restoreMetadataHash(metalId), 'compositeHash', 'hash256')];
```

Comment:
i changed this line 
```js
async binDataByMetalId(metalId) {
	const metadatasByCompositeHash = await this.metadatasByCompositeHash([metal.extractCompositeHashFromMetalId(metalId)]);
```
because extractCompositeHashFromMetalId return typeof uint8array